Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/conflict.py

```diff
@@ -84,11 +84,32 @@ def main(cls, args, repository, **kwargs):
         radar_id = ''.join(i for i in args.radar if i.isdigit())
         radar_obj = Tracker.from_string(f'rdar://{radar_id}')
         expected_branch = 'integration/conflict/{}'.format(radar_obj.id)
+        conflict_pr = None
+        source_remote = None
         for source_remote in repository.source_remotes():
             rmt = repository.remote(name=source_remote)
             conflict_pr = cls.find_conflict_pr(rmt, radar_obj.id)
             if conflict_pr:
-                return repository.checkout('{}:{}'.format(conflict_pr._metadata['full_name'], conflict_pr.head))
+                break
+
+        if not conflict_pr:
+            sys.stderr.write('No conflict pull request found with branch {}\n'.format(expected_branch))
+            return 1
+
+        full_branch = '{}:{}'.format(conflict_pr._metadata['full_name'], conflict_pr.head)
+        print('Found conflict branch {}'.format(full_branch))
+        repository.checkout(full_branch)
+
+        msg = "\n\n-------------------------------------------------------"
+        msg += "\nYou are now checked out into the conflict branch.\n"
+        msg += "Conflict markers are present in files.\n"
+        msg += "Please resolve them, amend the commit and force push.\n"
+        msg += "\nAlternatively, if you want to get into the traditional conflict state (ex. `git status` shows conflicting files)\n"
+        msg += "you can run the following commands. Note: This will remove any changes that were edited on the branch directly."
+        msg += f'\n\ngit reset --hard {source_remote}/{conflict_pr.base}\n'
+        for source in radar_obj.source_changes:
+            source_sha = source.split(', ')[2]
+            msg += f'git cherry-pick {source_sha}\n'
```

Comment:
I think f-string is a no-go in this codebase but I could be wrong

Reply:
f-strings should be fine since 5200ed9 when we started mandating Python 3.6